docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
```

Comment:
What version of yarn?

Reply:
I don't think folks are usually pinning the version of yarn to use, similar to `npm`

Reply:
Are you sure `1.22` doesn't work if you're in the docs directory? I've been running it with `npm`, but I have `yarn@1.22` as well, and `cd docs && yarn start` runs without issue. Can you expand on the errors you ran into? 

Docusaurus itself doesn't have any tooling requirements. Rather than put arbitrary version requirements here, we can just deeplink to the docusaurus documentation